web_src/js/features/repo-commit.js

```diff
@@ -57,7 +57,7 @@ export function initRepoCommitLastCommitLoader() {
 }
 
 export function initCommitStatuses() {
-  $('.commit-statuses-trigger').each(function () {
+  $('[data-tippy="commit-statuses"]').each(function () {
```

Comment:
In case you want to do want to get away from jQuery:

```js
document.querySelectorAll('[data-tippy="commit-statuses"]').forEach(function () {
  /* … */
}
```

should do the same.

Reply:
I don’t really want to go down the rabbit hole of removing jquery now…